modin/pandas/test/dataframe/test_join_sort.py

```diff
@@ -450,10 +450,28 @@ def test_sort_values(
     if ascending is None and key is not None:
         pytest.skip("Pandas bug #41318")
 
+    # If index is preserved, it could be sorted differently from Pandas.
+    # The order of NA rows, sorted by HDK, is different (but still valid)
+    # from Pandas. To make the index identical to Pandas, we add the
```

Comment:
could you please provide an example for this?

Reply:
For example:

```python
idx = pandas.Index(list("ABCD"))
data = {"A": [None, None, 2, 1]}
print(pandas.DataFrame(data, index=idx).sort_values(by="A"))
print(pd.DataFrame(data, index=idx).sort_values(by="A"))
```

Output:
```
     A
D  1.0
C  2.0
A  NaN
B  NaN

     A
D  1.0
C  2.0
B  NaN
A  NaN
```

Both results seem valid, but the order of NaN rows is different.